tests/unit/ForeignJurisdiction.spec.ts

```diff
@@ -29,80 +29,124 @@ describe('ForeignJurisdiction', () => {
       })
     const vm: any = wrapper.vm
 
-    wrapper.find('#country-selector').trigger('click')
-    await Vue.nextTick()
-    wrapper.find('.menuable__content__active').findAll('.v-list-item').at(123).trigger('click')
+    await wrapper.find('#country-selector').trigger('click')
+    wrapper.findAll('.v-list-item').at(123).trigger('click')
 
     expect(vm.selectedCountry).toEqual({code: 'LB', name: 'Lebanon'})
     expect(vm.selectedRegion).toEqual({})
     wrapper.destroy()
   })
 
-  xit('set correct region when selected via the region selector', async () => {
+  it('set correct Canadian province when selected via the region selector', async () => {
     const wrapper = mount(ForeignJurisdiction,
       {
         vuetify
       })
     const vm: any = wrapper.vm
 
-    wrapper.find('#country-selector').trigger('click')
-    await Vue.nextTick()
-    wrapper.findAll('.v-list-item').at(39).trigger('click')
+    await wrapper.find('#country-selector').trigger('click')
+    await wrapper.findAll('.v-list-item').at(39).trigger('click')
+
+    const region = wrapper.find('#region-selector')
+    await region.setValue(JSON.stringify({code: 'ON', name: 'Ontario'}))
+
     expect(vm.selectedCountry).toEqual({code: 'CA', name: 'Canada'})
+    expect(vm.selectedRegion).toEqual(JSON.stringify({code: 'ON', name: 'Ontario'}))
```

Comment:
Why does this need to be stringified but the check on line 54 doesn't?

Reply:
I was running into all sorts of errors and problems since `v-select` works in mysterious ways. Basically, for line 53, it's OK since I'm setting the object via a click. For the region, when I try do that, I'm running into a problem where it's not setting it correctly. By using the `stringify` method, it fixed the problem.

I can show you more details if you want Sev regarding this. It's just I spent way too much time here than I would like 😟

Reply:
Are you able to "click" the region same as country?

Reply:
No! I've tried every single possible way ever. It doesn't work. Pretty sure since there are two v-selects. It's not getting the input as I had hoped for. I've searched everywhere but there isn't much out there. This is the solution I came up with.

Reply:
OK, leave it then. It's fine, just different.